minwoo.lee_java/src/BOJ2812.java

```diff
@@ -0,0 +1,30 @@
+import java.util.*;
+import java.io.*;
+
+public class BOJ2812 {
+    public static void main(String[] args) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        StringTokenizer st = new StringTokenizer(br.readLine());
+        int N = Integer.parseInt(st.nextToken());
+        int k = Integer.parseInt(st.nextToken());
+        int printCnt = N - k;
+        char[] nums = br.readLine().toCharArray();
+        Stack<Character> stack = new Stack<>();
+        stack.push(nums[0]);
+        for (int i = 1; i < nums.length; i++) {
+            while (!stack.isEmpty() && k != 0) {
+                if (stack.peek() >= nums[i]) {
+                    break;
+                }
```

Comment:
이 조건은 아래와 같이 쓸수도 있습니다.
```suggestion
            while (!stack.isEmpty() && k != 0 && stack.peek() < nums[i]) {
                ...
            }
```

Reply:
수정하였습니다. 감사합니다.